### ISSUE TITLE
Levels

### ISSUE THREAD

**Issue by hcarreras** [hcarreras/party-game#1]
Parties start slowly. No one dares to do anything at the beginning, but when the party is started, no one wants a soft challenge.

The feature could work this way:
The challenges are defined in the JSON by levels. Each level consists of 10 random questions.
Then, we just show "Level X", so everyone is aware that we are on the next level.

Perhaps, we could also show the level somewhere in the upper part of the screen.

**Comment by monicamatei** [comment]
It depends at what stage of the party the game is started. If people are already tipsy, they might want to be able to choose the level they want to start with. 

I suggest having 4 levels: easy, medium, hard, very hard. (we can even give then funnier names or descriptions such as 'sober - I need a slow start', 'tipsy - I am up for something more difficult', 'drunk - I can handle more!' and 'very drunk - I am up for any challenge'. -> though we can argue that, if people are very drunk, they won't play the game at all.

**Comment by hcarreras** [comment]
That’s a great idea. Sometimes you’d like to start with something a bit more challenging. We don’t want to discourage party people because it’s too slow.

I see that as 2 different features. One for having levels, the second for being able to start in a more advanced level.

We can already agree in the 4 levels.
We could even theme the game. In that case, the names could be related. For now, I’d roll with the names you wrote.

**Comment by hcarreras** [comment]
I made a PR: https://github.com/hcarreras/party-game/pull/5

@monicamatei this Pull Request only includes the first part of the feature, including levels. At the moment it's not possible to start in a different level. 
We don't have a menu yet, we probably need to add a menu before having the possibility to start at a certain level.

I took the short names for the levels bar. I think we should reserve the long names for the menu.

I think the names are very appropriate. The first levels should focus more on drinking and higher levels more on fun challenges. That way, people don't drink too much once they are already drunk.
